parcel.go

```diff
@@ -35,26 +56,68 @@ func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
-
+	rows, err := s.db.Query("SELECT number, status, address, created_at FROM parcel WHERE client = :client",
+		sql.Named("client", client))
+	if err != nil {
+		return res, nil
```

Comment:
Вообще что-то не понятное тут
```suggestion
		return nil, err
```

Reply:
экспериментировал)) исправил!